Scaffolding retreive contac

MarkMango21 opened the pull request:
So far this is what I had with retrieve contact functions and calls. I'm using another HTML that we could anchor in the index. Or if you guys have nay others ways to approach this plz lmk. I should mention that I will change some headers of the table to print the phone number and email related to the user

MarkMango21 committed:
Creation of the User and Contact classes

MarkMango21 committed:
Setters for Contact and user

Basic functions to set and create contact and user elements.

MarkMango21 committed:
Rename the php document and create API folder

MarkMango21 committed:
Change in the API doc login.php

after actually reviewing an understanding what does API takes care of. I created the folder API, and changed the login.php doc completely

MarkMango21 committed:
Create the user in Data

MarkMango21 committed:
Login php. functions structure

Still getting the POST calls for the Data base to log in as a user

MarkMango21 committed:
login.php file progress

MarkMango21 committed:
Commented  on login.php file

Made some little progress in the php file.

MarkMango21 committed:
Little update login.php

MarkMango21 committed:
Clean the login.php doc

MarkMango21 committed:
New doc Retreive contact

Start the work on retreiving the contacts once the user log in

MarkMango21 committed:
Login doc update

MarkMango21 committed:
Retreive Contacs

Hi, I made more progress on the Retreive contacs HTML, and I though of using another HTML to print the user's contact .
I will tomorrow clear and make it more logical. Like adding the contact numbers and emails into the table

Teesitin reviewed (approved):
Needs some styling. Other then that, looks g u c c i

Teesitin reviewed (changes requested):
Looks like we have some conflicts. Let's review together asap

MarkMango21 commented:
Sounds good to me. We can work on it tomorrow. Since I have to get ready and get some sleep for class. You can hit me up in discord and we can discuss it.

MarkMango21 committed:
Some changes in the retreive call

MarkMango21 committed:
Login.php and Retreive logic done

MarkMango21 committed:
Menu php document

I though, logically to add a menu php file to promp the ohter php files so we depending on what the user does it takes you to the php file for the action.

MarkMango21 committed:
Menu php doc changed
